Docker.DotNet/Microsoft.Net.Http.Client/UnixDomainSocketEndPoint.cs

```diff
@@ -0,0 +1,110 @@
+#if NETSTANDARD1_6
```

Comment:
I would... but VSCode doesn't like explicit excludes in the project.json.  The empty file isn't the end of the world...